src/Sarif/Writers/PrereleaseCompatibilityTransformer.cs

```diff
@@ -122,9 +126,32 @@ public static class PrereleaseCompatibilityTransformer
             return transformedSarifLog;
         }
 
+        private static bool ApplyChangesFromTC32(JObject sarifLog)
+        {
+            UpdateSarifLogVersion(sarifLog);
+            if (sarifLog["runs"] is JArray runs)
+            {
+                foreach (JObject run in runs)
+                {
+                    // https://github.com/oasis-tcs/sarif-spec/issues/325
+                    RemoveToolLanguage(run);
+                }
+            }
+            return true;
+        }
+
+        private static void RemoveToolLanguage(JObject run)
+        {
+            JObject tool = (JObject)run["tool"];
+            if (tool["language"] is JToken language)
+            {
+                tool.Remove("language");
+            }
+        }
+
         private static bool ApplyChangesFromTC31(JObject sarifLog)
         {
-            UpdateSarifLogVersion(sarifLog);            ;
+            UpdateSarifLogVersion(sarifLog);
```

Comment:
>UpdateSarifLogVersion(sarifLog); [](start = 12, length = 32)

we can remove this call now that we have it in the TC32 helper. I'll do this in the next change!